doc/user/content/sql/create-table.md

```diff
@@ -1,31 +1,42 @@
 ---
 title: "CREATE TABLE"
-description: "`CREATE TABLE` creates a non-streaming, in-memory data source."
+description: "`CREATE TABLE` creates an in-memory table."
 menu:
   main:
     parent: 'sql'
 ---
 
 {{< version-added v0.5.0 >}}
 
-`CREATE TABLE` creates a non-streaming, in-memory data source.
+`CREATE TABLE` creates an in-memory table.
 
-{{< warning >}}
-At the moment, tables do not persist any data that is inserted. This means that restarting a
-Materialize instance will lose any data that was previously stored in a table.
-{{< /warning >}}
+## Conceptual framework
 
+Tables in Materialize are similar to tables in standard relational databases:
+they consist of rows and columns where the columns are fixed when the table is
+created but rows can be added to at will via [INSERT](../insert) statements.
 
-## Conceptual framework
+You can seamlessly join tables with other tables, views, and sources in the
+system.
 
-Tables store non-streaming data that is inserted via [INSERT](../insert) statements.
+{{< warning >}}
+At the moment, tables serve a niche use case. They are lacking many features
+that are standard in relational databases. In most situations you should use
+[sources](/sql/create-source) instead.
+{{< /warning >}}
 
-### When to create a table
+### When to use a table
 
-You might want to create a table when...
+Using a table can be more convenient than using a source, but only if all of
+the following statements are true:
 
-- Manually inserting rows of data into Materialize.
-- Testing Materialize's features without setting up a data stream.
+1. Your dataset is either static or append only.
```

Comment:
append-only (hyphenated)

Reply:
This one I left as is because I was always taught that compound adjectives only need to be hyphenated when they appear as attributive adjectives, not as a predicative adjectives.